tests/test_multi_agent_system.py

```
@@ -0,0 +1,593 @@
+            await asyncio.sleep(5)  # Simulate slow operation
+            return "Slow result"
+        
+        researcher.generate_research = slow_operation
+        
+        # Test timeout handling
+        try:
+            result = await asyncio.wait_for(
+                researcher.generate_research("Test Topic"),
+                timeout=2.0
+            )
+            assert False, "Should have timed out"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)